src/client/index.js

```diff
@@ -11,6 +11,7 @@
 import { useState, useEffect, useContext, createContext, createElement } from 'react'
```

Comment:
I have to go through the client code more thoroughly, but I don't think we should make it configurable by the client. I actually think we shouldn't make this configurable at all if possible, we should just follow the language Next.js provides. Haven't used Next.js's i18n features yet, so I'm not totally sure just yet if that is possible. Will check.

Reply:
The implementation I've got going here uses the locale provided by the Next.js router & assumes that it is not changed manually, so yes agree on that front.